doctr/datasets/vocabs.py

```diff
@@ -73,6 +75,7 @@
         + VOCABS["danish"]
         + VOCABS["finnish"]
         + VOCABS["swedish"]
+        + VOCABS["ukrainian"]
```

Comment:
Please remove from the multilingual entry because we don't have a multilingual dataset which includes cyrillic letters atm :)